docs/content/Getting-Started/user-quickstart-openshift.md

```diff
@@ -0,0 +1,243 @@
+        Stuck? [open a bug report and we can help you out](https://github.com/kubestellar/kubestellar/issues/new?assignees=&labels=kind%2Fbug&projects=&template=bug_report.yaml&title=bug%3A+)
+
+
+#### 4. Install <span class="Space-Bd-BT">KUBESTELLAR</span> Syncers on your Edge Clusters
+!!! tip ""
+    === "Prep and apply"
+        prepare <span class="Space-Bd-BT">KUBESTELLAR</span> Syncers, with `kubestellar prep-for-cluster`, for **ks-edge-cluster1** and **ks-edge-cluster2** and then apply the files that `kubestellar prep-for-cluster`` prepared for you
+
+        **important:** make sure you created Kind clusters for **ks-edge-cluster1** and **ks-edge-cluster2** from the pre-req step above before proceeding [how-to-deploy-and-use-kubestellar](#how-to-deploy-and-use-kubestellar)
+
+         {%
+           include-markdown "../common-subs/kubestellar-prep-syncer.md"
+           start="<!--kubestellar-prep-syncer-start-->"
+           end="<!--kubestellar-prep-syncer-end-->"
+         %}
+
+         {%
+           include-markdown "../common-subs/kubestellar-apply-syncer.md"
+           start="<!--kubestellar-apply-syncer-start-->"
+           end="<!--kubestellar-apply-syncer-end-->"
+         %}
+
+#### 5. Deploy an Apache Web Server to ks-edge-cluster1 and ks-edge-cluster2
+!!! tip ""
+    === "deploy"
+         {%
+           include-markdown "../common-subs/kubestellar-apply-apache.md"
+           start="<!--kubestellar-apply-apache-start-->"
+           end="<!--kubestellar-apply-apache-end-->"
+         %}
+
+#### 6. View the Apache Web Server running on ks-edge-cluster1 and ks-edge-cluster2
+!!! tip ""
+    === "view"
+         {%
+           include-markdown "../common-subs/kubestellar-test-apache.md"
+           start="<!--kubestellar-test-apache-start-->"
+           end="<!--kubestellar-test-apache-end-->"
+         %}
+    === "uh oh, error?"
+         {%
+           include-markdown "../common-subs/kubestellar-check-syncers.md"
+           start="<!--kubestellar-check-syncers-start-->"
+           end="<!--kubestellar-check-syncers-end-->"
+         %}
+
+#### 7. Check the status of your Apache Server on ks-edge-cluster1 and ks-edge-cluster2
+
+```
+TODO
+```
+
+what's next...  
+how to upsync a resource  
+how to create, but not overrite/update a synchronized resource  
+
+<br>
+---
+
+---
+<br>
+
+## How to use an existing <span class="Space-Bd-BT">KUBESTELLAR</span> environment
+
+#### 1. Install KubeStellar's user commands and kubectl plugins
+!!! tip ""
+    === "install"
+         {%
+           include-markdown "../common-subs/brew-install.md"
+           start="<!--brew-install-start-->"
+           end="<!--brew-install-end-->"
+         %}
+    === "remove"
+         {%
+           include-markdown "../common-subs/brew-remove.md"
+           start="<!--brew-remove-start-->"
+           end="<!--brew-remove-end-->"
+         %}
+
+#### 2. View your <span class="Space-Bd-BT">KUBESTELLAR</span> Core Space environment
```

Comment:
This section also does something much more impactful that listing some workspaces: it fetches the external kubeconfig and puts in a particular file. This should be called out in the section title as well as getting explicit mention in the English text.